app/controllers/api/v1/jobs/show.rb

```diff
@@ -0,0 +1,23 @@
+module API
+  module V1
+    module Jobs
+      class Show < Base
+        params do
+          requires :id, type: Integer, desc: 'Id of a certain job'
+        end
+
+        desc 'Return a job'
+        get '/:id' do
+          @job = Job.find_by(id: params[:id])
```

Comment:
I would suggest creating before block in base API and. Also if you are going to have couple API endpoints which require `:id` and it has to be passed in an url you can create route_param. Check Grape documentation for more details.